package.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/sprjr/react-access#readme",
   "files": ["dist"],
   "peerDependencies": {
-    "react": "0.14.x || 15.3.x"
+    "react": "0.14.x || 15.x"
```

Comment:
This actually causes the tests to break, as there is an issue with `15.4.x` branch with the combination of enzyme, react-addons-test, react, and react-dom, which is why I went a bit more specific here initially. I was planning on trying to resolve that when I get the remainder of the tests written. If you update this PR to also include `.travis.yml` changes for a React 15.x version, you should see the errors output there.